### ISSUE TITLE
Messed up terrain gen with The Hallow

### ISSUE THREAD

**Issue by spyguy26** [valoeghese/TBOClassic#1]
when playing with both the hallow and TBO-classic world gen seems to turn striped between hallow biomes and vanilla biomes as pictured in
![2019-11-12_18 16 22](https://user-images.githubusercontent.com/39753561/68719641-99a38180-057a-11ea-98dd-359528f1548a.png)

to reproduce have both The Hallow and tbo-classic, create a new world (as worlds created without tbo-classic are unaffected) and enter the hallow

fabric loader version: 0.6.4+build.169
fabric api version: fabric-api-0.4.1+build.245-1.14
The Hallow version: TheHallow-1.1.0-beta+1.14.4
Trinkets version: trinkets-2.1.1
Tbo-classic version: tbo-classic-1.0.0

**Comment by valoeghese** [comment]
Striped terrain like that usually indicates some sort of byte overflow

**Comment by valoeghese** [comment]
In which case it just means there are too many biomes in the game...
Because for some reason mojang still doesn’t have biome ids over 255 working properly

**Comment by valoeghese** [comment]
Do you get “unknown biome id” spammed in the game log while playing?

**Comment by spyguy26** [comment]
not seeing anything in the log about unknown biome id

**Comment by valoeghese** [comment]
Yeah that message usually doesn't stay in the log when it happens I think

**Comment by valoeghese** [comment]
Anyway this issue isn't the fault of TBOClassic as far as I can tell. It's most likely to do with how mojang handles biome ids over 255
